### PR TITLE
fix map-info-box display

### LINKED ISSUE #791: Livemap: Map info box does not appear after clicking on a plane
**Describe the bug**
When clicking on a plane in the livemap widget, the map info box (transparent gray div on the bottom portion of the map that displays flight/pirep-specific information) does not appear. The class "map-info-box" has a display: none CSS property that is not cleared when the onClick event is triggered.

**Version**
v7.0.0.dev-latest-build

**To Reproduce**
1. Go to /livemap
2. Click on a plane icon that represents a live flight
3. Observe that the flight-specific box does not appear

**Expected behavior**
The map info box should appear when a flight/plane is clicked on.

**Screenshots**
Expected behavior:

Actual Result:

### PR DESCRIPTION
**Pull request by exciler**
fixes #791

### CODE REVIEW

**Comment by FatihKoz** [comment]
Would you please add pireps\show.blade.php to this pr and close a possible problem before this becomes live ?

Please do change pireps\show.blade.php `@if(!$pirep->read_only)` to this `@if(!$pirep->read_only && $pirep->user_id === Auth::id())`

Cause when we click on the map info box flight number, it opens the pirep and since it is still in progress, the edit and submit buttons are visible. We need to show that buttons only to the owner ;)

Thanks

File: resources/views/layouts/default/pireps/show.blade.php [hunk 1]
```diff
@@ -15,7 +15,7 @@
            class="btn btn-outline-info">View SimBrief</a>
       @endif
 
-      @if(!$pirep->read_only)
+      @if(!$pirep->read_only && $pirep->user_id === Auth::id())
```
Comment (nabeelio) [inline resources/views/layouts/default/pireps/show.blade.php:18]: One thing - in the controller for the view, set a `user` and then here use `$pirep->user_id === $user->id`. I'd rather the $user be set in the controllers.

File: app/Http/Controllers/Frontend/PirepController.php [hunk 2]
```diff
@@ -491,6 +496,9 @@ public function update($id, UpdatePirepRequest $request)
             Flash::error('Pirep not found');
             return redirect(route('admin.pireps.index'));
         }
+        if ($pirep->user_id !== Auth::id()) {
```
Comment (nabeelio) [inline app/Http/Controllers/Frontend/PirepController.php:499]: Good call on this... I thought I had put them in the `UpdatePirepRequest` classes but apparently, I only did that for the ACARS requests. So I think this should actually go into `App/Http/Requests/UpdatePirepRequest`, there should be a 'authorize()` method. Take a look at `App/Http/Requests/Acars/EventRequest`. 

I think something like

```php
$pirepRepo = app(PirepRepository::class); // import it
$pirep = $pirepRepo->find($this->route('pirep_id'));
return $pirep->user_id === Auth::id();
```

The other ones are fine, though. Thanks!

Comment (exciler) [reply app/Http/Controllers/Frontend/PirepController.php:499]: Thank you for pointing me there. Those request classes were new to me. I fully agree

**Pull request merged**